### PR TITLE
NH-73496 Update layer exec wrapper

### PR DESCRIPTION
**Pull request by tammy-baylis-swi**
Changes layer exec wrapper entrypoint so that `AWS_LAMBDA_EXEC_WRAPPER: /opt/solarwinds-apm/wrapper`. See [comment](https://swicloud.atlassian.net/browse/NH-73496?focusedCommentId=2624406) for some tests.

### CODE REVIEW

**Comment by tammy-baylis-swi** [comment]
Need to make one change to attributes.

File: lambda/solarwinds-apm/wrapper [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+fi
+
+# Lambda function is default service name
+if [ -z "${OTEL_SERVICE_NAME}" ]; then
+    export OTEL_SERVICE_NAME=$AWS_LAMBDA_FUNCTION_NAME;
+fi
+
+# Set faas.instance resource attribute to expose log stream name.
+# Currently we ignore any configured OTEL_RESOURCE_ATTRIBUTES.
+export OTEL_RESOURCE_ATTRIBUTES="faas.instance=$AWS_LAMBDA_LOG_STREAM_NAME";
```
Comment (tammy-baylis-swi) [inline lambda/solarwinds-apm/wrapper:42]: Let me know if this should append to any `OTEL_RESOURCE_ATTRIBUTES` if set

File: lambda/solarwinds-apm/wrapper [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+if [ -z "${OTEL_EXPORTER_OTLP_PROTOCOL}" ]; then
+    export OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+fi
+
+# Lambda function is default service name
+if [ -z "${OTEL_SERVICE_NAME}" ]; then
+    export OTEL_SERVICE_NAME=$AWS_LAMBDA_FUNCTION_NAME;
+fi
+
+# Set faas.instance resource attribute to expose log stream name.
```
Comment (cheempz) [inline lambda/solarwinds-apm/wrapper:40]:
```suggestion
# Set faas.instance resource attribute from the log stream name.
```

File: lambda/solarwinds-apm/wrapper (hunk 1 above)
Comment (cheempz) [reply lambda/solarwinds-apm/wrapper:42]: Good question! I think we should, in the case that the customer sets `OTEL_RESOURCE_ATTRIBUTES` env var on their AWS Lambda Function config to add custom attributes--which we should respect and just pass along.

**Review by cheempz — COMMENTED** [review]
Appreciate the test data as usual @tammy-baylis-swi!

File: lambda/solarwinds-apm/wrapper (hunk 2 above)
Comment (tammy-baylis-swi) [reply lambda/solarwinds-apm/wrapper]: Addressed in [144b1dd](https://github.com/solarwinds/apm-python/pull/317/commits/144b1dd918e6733fafb5755daccac8309a67e4a0)

File: lambda/solarwinds-apm/wrapper (hunk 1 above)
Comment (tammy-baylis-swi) [reply lambda/solarwinds-apm/wrapper]: Addressed in [fec5ba0](https://github.com/solarwinds/apm-python/pull/317/commits/fec5ba0187780150c7447370ebfd8360fe760f0c)!

**Comment by tammy-baylis-swi** [comment]
Thanks for first review @cheempz ! Please let me know if anything else.

**Review by cheempz — APPROVED** [review]
LGTM, thanks @tammy-baylis-swi!

**Pull request merged**